### DISCUSSION TITLE
Is there anyone installing latex-yearly-planner project succesfully?

### DISCUSSION THREAD

**Discussion by nmduc073** [kudrykv/latex-yearly-planner#59] [General]
I am trying to rebuild latex-yearly-planner project. I want to change the 2023 version. I would like it to look lile 2022 version. In the 2023 version, the length of daily schedule is too short while top priorites is too long.

However, there is an error when I run [single.sh](https://single.sh/) in cmd.

[main 2022-10-01T10:28:28.171Z] update#setState idle

[main 2022-10-01T10:28:29.446Z] Starting extension host with pid 7428 (fork() took 14 ms).

[main 2022-10-01T10:28:58.180Z] update#setState checking for updates

[main 2022-10-01T10:28:58.219Z] update#setState idle

[8796:1001/173027.972:ERROR:gpu_init.cc(481)] Passthrough is not supported, GL is disabled, ANGLE is

Please instruct me to rebuild this project. Thank in advance.

**Comment by kudrykv** [comment]
Hi @nmduc073!

That's are weird errors for sure!

Make sure you have the latest Go and latex installed. The `single.sh` requires a bunch of env variables defined.
For example, to build an MOS version, you need to run the such command:
```shell
PLANNER_YEAR=2023 \
PASSES=2 \
CFG="cfg/base.yaml,cfg/template_months_on_side.yaml,cfg/sn_a5x.mos.default.yaml,cfg/sn_a5x.mos.default.dailycal.yaml" \
NAME=sn_a5x.mos.default.dailycal.2023 \
./single.sh
```

`CFG` is a comma-separated list of configurations. They got merged into a single one; the one on the right has higher priority than the one on the left in terms of overriding values.

**Reply by nmduc073** [reply]
Thank for your reply,
I installed on my laptop. I followed your instruction on Install Dependencies section. It must be the latest version.
In the cmd, I go to the project directory.
C:\Users\Thinkpad>cd C:\Yamicsoft\latex-yearly-planner-main

C:\Yamicsoft\latex-yearly-planner-main>single.sh

C:\Yamicsoft\latex-yearly-planner-main>
[main 2022-10-02T09:46:56.325Z] update#setState idle
[main 2022-10-02T09:46:59.256Z] Starting extension host with pid 6180 (fork() took 43 ms).
update#setState checking for updates
[main 2022-10-02T09:47:26.416Z] update#setState idle
...
I didn't change anything in single.sh file or any file. Do I have to change anything in that file?

**Reply by nmduc073** [reply]
Sorry because I am newbie to LaTex.
I don't know where to run this code? If I run it in the cmd, there is error code
PLANNER_YEAR=2023 \
PASSES=2 \
CFG="cfg/base.yaml,cfg/template_months_on_side.yaml,cfg/sn_a5x.mos.default.yaml,cfg/sn_a5x.mos.default.dailycal.yaml" \
NAME=sn_a5x.mos.default.dailycal.2023 \
./single.sh


C:\Yamicsoft\latex-yearly-planner-main>PLANNER_YEAR=2023 \
'PLANNER_YEAR' is not recognized as an internal or external command,
operable program or batch file.

**Reply by kudrykv** [reply]
Oh, you're on Windows. That certainly complicates things. I guess that's why you're getting those errors: in Windows, you have different commands.

You would need to run commands from the `single.sh` manually, so to speak, modifying them for Windows.

First, run Go to create tex files:
```shell
set PLANNER_YEAR=2023
go run cmd/plannergen/plannergen.go --config "cfg/base.yaml,cfg/template_months_on_side.yaml,cfg/sn_a5x.mos.default.yaml,cfg/sn_a5x.mos.default.dailycal.yaml"
```

Then, build PDF with `pdflatex` (for MOS, run this command twice):
```shell
pdflatex out/document.tex
```

You need to have `go` and `pdflatex` in your `PATH` for those commands to work

**Reply by kudrykv** [reply]
> I don't know where to run this code

Yeah, that's on cmd or powershell. Still, those commands don't apply one-to-one for Windows. `single.sh` won't work in Windows at all.

**Reply by nmduc073** [reply]
Thank for your help. I passed to pdflatext command
C:\Yamicsoft\latex-yearly-planner-main>go run cmd/plannergen/plannergen.go --config "cfg/base.yaml,cfg/template_months_on_side.yaml,cfg/sn_a5x.mos.default.yaml,cfg/sn_a5x.mos.default.dailycal.yaml"

C:\Yamicsoft\latex-yearly-planner-main>pdflatex out/document.tex
This is pdfTeX, Version 3.141592653-2.6-1.40.24 (MiKTeX 22.8.28) (preloaded format=pdflatex.fmt)
 restricted \write18 enabled.
entering extended mode
! I can't find file `out/document.tex'.
<*> out/document.tex

(Press Enter to retry, or Control-C to exit)
Please type another input file name:
! I can't find file `out/document.tex'.
<*> out/document.tex

(Press Enter to retry, or Control-C to exit)
Please type another input file name: out/monthly.tex
(out/monthly.tex
LaTeX2e <2022-06-01> patch level 5
L3 programming layer <2022-08-30>

! LaTeX Error: Missing \begin{document}.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.

**Reply by kudrykv** [reply]
Hm. After running `go run ...`, did you get files in the `out` directory?

**Reply by nmduc073** [reply]
Yes, there are some file
annual.tex
daily.tex
daily_notes.tex
daily_reflect.tex
monthly.tex
notes_indexed.tex
quarterly.tex
sn_a5x.mos.default.dailycal.tex
title.tex
weekly.tex

**Reply by nmduc073** [reply]
My mistake. I should use the command
C:\Yamicsoft\latex-yearly-planner-main>pdflatex out/sn_a5x.mos.default.dailycal.tex
This is pdfTeX, Version 3.141592653-2.6-1.40.24 (MiKTeX 22.8.28) (preloaded format=pdflatex.fmt)
 restricted \write18 enabled.
entering extended mode
(out/sn_a5x.mos.default.dailycal.tex
LaTeX2e <2022-06-01> patch level 5
L3 programming layer <2022-08-30>
======================================================================
starting package maintenance...
installation directory: D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX
package repository: https://mirror.kku.ac.th/CTAN/systems/win32/miktex/tm/packages/
visiting repository https://mirror.kku.ac.th/CTAN/systems/win32/miktex/tm/packages/...
repository type: remote package repository

**Reply by kudrykv** [reply]
That's good
Then, run `pdflatex sn_a5x.mos.default.dailycal.tex` or (`pdflatex out/sn_a5x.mos.default.dailycal.tex`—depending where you are) and see if that builds sn_a5x.mos.default.dailycal.pdf for you.
You need to run the command twice because MOS is tricky, and here latex requires files it builds on the first run to produce the correct result on the second one.

**Reply by nmduc073** [reply]
C:\Yamicsoft\latex-yearly-planner-main>pdflatex out/sn_a5x.mos.default.dailycal.tex
This is pdfTeX, Version 3.141592653-2.6-1.40.24 (MiKTeX 22.8.28) (preloaded format=pdflatex.fmt)
 restricted \write18 enabled.
entering extended mode
(out/sn_a5x.mos.default.dailycal.tex
LaTeX2e <2022-06-01> patch level 5
L3 programming layer <2022-08-30>
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/extsizes\extarticle.
cls
Document Class: extarticle 1996/10/08 v1.0 Non Standard LaTeX document class
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/extsizes\size9.clo)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/base\exscale.sty))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/geometry\geometry.st
y
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/graphics\keyval.sty)
 (D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/iftex\ifvtex.sty
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/iftex\iftex.sty))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/geometry\geometry.cf
g))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/xcolor\xcolor.sty
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/graphics-cfg\color.c
fg)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/graphics-def\pdftex.
def)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/colortbl\colortbl.st
y (D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/tools\array.sty))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex-dev/graphics\mathcol
or.ltx))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/tools\calc.sty)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/dashrule\dashrule.st
y
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/ifmtarg\ifmtarg.sty)
)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/setspace\setspace.st
y)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/pgf/frontendlayer\ti
kz.sty
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/pgf/basiclayer\pgf.s
ty
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/pgf/utilities\pgfrcs
.sty
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/utilities\pgfu
til-common.tex
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/utilities\pgfu
til-common-lists.tex))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/utilities\pgfu
til-latex.def)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/utilities\pgfr
cs.code.tex
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf\pgf.revision.t
ex)))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/pgf/basiclayer\pgfco
re.sty
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/graphics\graphicx.st
y
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/graphics\graphics.st
y (D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/graphics\trig.sty)

(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/graphics-cfg\graphic
s.cfg)))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/pgf/systemlayer\pgfs
ys.sty
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/systemlayer\pg
fsys.code.tex
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/utilities\pgfk
eys.code.tex
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/utilities\pgfk
eysfiltered.code.tex))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/systemlayer\pg
f.cfg)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/systemlayer\pg
fsys-pdftex.def
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/systemlayer\pg
fsys-common-pdf.def)))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/systemlayer\pg
fsyssoftpath.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/systemlayer\pg
fsysprotocol.code.tex))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/basiclayer\pgf
core.code.tex
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/math\pgfmath.c
ode.tex
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/math\pgfmathca
lc.code.tex
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/math\pgfmathut
il.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/math\pgfmathpa
rser.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/math\pgfmathfu
nctions.code.tex
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/math\pgfmathfu
nctions.basic.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/math\pgfmathfu
nctions.trigonometric.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/math\pgfmathfu
nctions.random.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/math\pgfmathfu
nctions.comparison.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/math\pgfmathfu
nctions.base.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/math\pgfmathfu
nctions.round.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/math\pgfmathfu
nctions.misc.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/math\pgfmathfu
nctions.integerarithmetics.code.tex)))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/math\pgfmathfl
oat.code.tex))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/math\pgfint.co
de.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/basiclayer\pgf
corepoints.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/basiclayer\pgf
corepathconstruct.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/basiclayer\pgf
corepathusage.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/basiclayer\pgf
corescopes.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/basiclayer\pgf
coregraphicstate.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/basiclayer\pgf
coretransformations.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/basiclayer\pgf
corequick.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/basiclayer\pgf
coreobjects.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/basiclayer\pgf
corepathprocessing.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/basiclayer\pgf
corearrows.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/basiclayer\pgf
coreshade.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/basiclayer\pgf
coreimage.code.tex
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/basiclayer\pgf
coreexternal.code.tex))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/basiclayer\pgf
corelayers.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/basiclayer\pgf
coretransparency.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/basiclayer\pgf
corepatterns.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/basiclayer\pgf
corerdf.code.tex)))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/modules\pgfmod
uleshapes.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/modules\pgfmod
uleplot.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/pgf/compatibility\pg
fcomp-version-0-65.sty)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/pgf/compatibility\pg
fcomp-version-1-18.sty))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/pgf/utilities\pgffor
.sty
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/pgf/utilities\pgfkey
s.sty
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/utilities\pgfk
eys.code.tex))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/pgf/math\pgfmath.sty

(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/math\pgfmath.c
ode.tex))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/utilities\pgff
or.code.tex
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/math\pgfmath.c
ode.tex)))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/frontendlayer/
tikz\tikz.code.tex
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/libraries\pgfl
ibraryplothandlers.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/modules\pgfmod
ulematrix.code.tex)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pgf/frontendlayer/
tikz/libraries\tikzlibrarytopaths.code.tex)))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/varwidth\varwidth.st
y)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/blindtext\blindtext.
sty (D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/tools\xspace.sty
))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/tools\tabularx.sty)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/wrapfig\wrapfig.sty)

(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/makecell\makecell.st
y)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/multirow\multirow.st
y)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/amsfonts\amssymb.sty

(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/amsfonts\amsfonts.st
y))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/l3kernel\expl3.sty
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/l3backend\l3backend-
pdftex.def))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/leading\leading.sty)

(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/hyperref\hyperref.st
y
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/ltxcmds\ltxcmds.st
y)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pdftexcmds\pdftexc
mds.sty
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/infwarerr\infwarer
r.sty))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/kvsetkeys\kvsetkey
s.sty)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/kvdefinekeys\kvdef
inekeys.sty)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/pdfescape\pdfescap
e.sty)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/hycolor\hycolor.sty)

(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/letltxmacro\letltxma
cro.sty)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/auxhook\auxhook.sty)

(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/hyperref\nameref.sty

(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/refcount\refcount.st
y)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/gettitlestring\get
titlestring.sty
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/kvoptions\kvoptions.
sty)))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/hyperref\pd1enc.def)

(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/intcalc\intcalc.st
y)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/etexcmds\etexcmds.
sty)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/hyperref\puenc.def)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/url\url.sty)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/bitset\bitset.sty
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/bigintcalc\bigintc
alc.sty))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/base\atbegshi-ltx.st
y))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/hyperref\hpdftex.def

(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/base\atveryend-ltx.s
ty)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/rerunfilecheck\rerun
filecheck.sty
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/uniquecounter\uniq
uecounter.sty)))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/marginnote\marginnot
e.sty)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/adjustbox\adjustbox.
sty
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/xkeyval\xkeyval.sty
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/xkeyval\xkeyval.te
x
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/xkeyval\xkvutils.t
ex)))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/adjustbox\adjcalc.st
y)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/adjustbox\trimclip.s
ty
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/collectbox\collectbo
x.sty)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/adjustbox\tc-pdftex.
def))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/ifoddpage\ifoddpage.
sty))
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/multido\multido.sty
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/generic/multido\multido.te
x  v1.42, 2010/05/14 <tvz>))

Package hyperref Warning: Unexpected value for option `hidelinks'
(hyperref)                is ignored on input line 27.

(sn_a5x.mos.default.dailycal.aux (title.aux) (annual.aux) (quarterly.aux)
(monthly.aux) (weekly.aux) (daily.aux) (daily_reflect.aux) (daily_notes.aux)
(notes_indexed.aux))
*geometry* driver: auto-detecting
*geometry* detected driver: pdftex

(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/context/base/mkii\supp-pdf
.mkii
[Loading MPS to PDF converter (version 2006.09.02).]
)
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/epstopdf-pkg\epstopd
f-base.sty
(D:\Users\Thinkpad\AppData\Local\Programs\MiKTeX\tex/latex/00miktex\epstopdf-sy
s.cfg)) (sn_a5x.mos.default.dailycal.out) (sn_a5x.mos.default.dailycal.out)
No file title.tex.
No file annual.tex.
No file quarterly.tex.
No file monthly.tex.
No file weekly.tex.
No file daily.tex.
No file daily_reflect.tex.
No file daily_notes.tex.
No file notes_indexed.tex.
(sn_a5x.mos.default.dailycal.aux (title.aux) (annual.aux) (quarterly.aux)
(monthly.aux) (weekly.aux) (daily.aux) (daily_reflect.aux) (daily_notes.aux)
(notes_indexed.aux)) )
No pages of output.
Transcript written on sn_a5x.mos.default.dailycal.log.

There is new file sn_a5x.mos.default.dailycal.pdf. But the size is 0 KB. I run twice and the output is the same.

**Reply by kudrykv** [reply]
> No file title.tex.
No file annual.tex.
No file quarterly.tex.
No file monthly.tex.
No file weekly.tex.
No file daily.tex.
No file daily_reflect.tex.
No file daily_notes.tex.
No file notes_indexed.tex

Try to change dir to `out` and run the command again. It looks that latex doesn't want to search for files relative to the original file.
```
cd out
pdflatex sn_a5x.mos.default.dailycal.tex
```

**Reply by kudrykv** [reply]
`dir out` if `cd` command doesn't work (I'm not sure what Windows has)

**Reply by nmduc073** [reply]
Thank you very much

**Comment by nmduc073** [comment]
I have another question.
What should I add to create rm2_ddvk.breadcrumb.lined.default.2023.pdf. The default is dotted.
go run cmd/plannergen/plannergen.go --config "cfg/base.yaml,cfg/template_breadcrumb.yaml,cfg/rm2.breadcrumb.default.yaml"

**Reply by kudrykv** [reply]
Those are what I'm using for start configurations: https://github.com/kudrykv/latex-yearly-planner/blob/e4b39ece1e0d26cf7fed8ac27ef951f2f007ed3b/release.sh#L7-L27

The first column is the number of runs (`pdflatex ...`). The second column is the config you need to pass into the go command. The comma-separated config are files that are layered one on each other to have one config. Those are being merged into one in the app; the file to the right wins in terms of the config value.

**Comment by nmduc073** [comment]
In rm2.base.yaml, I added:
dailynotes: 25
dailynotesmonthly: 10 --my code
In config.go, I added:
DailyNotes          int
	DailyNotesMonthly   int --my code
In marco.tpl, I added:
\newcommand{\myNumDailyNotes}{ {{- $numbers.DailyNotes -}} }
\newcommand{\myNumDailyNotesMonthly}{ {{- $numbers.DailyNotesMonthly -}} } --my code
But it doesn't work in monthly.tex:
{\Repeat{\myNumDailyNotes}{\myTodoLineGray}} --it works
{\Repeat{\myNumDailyNotesMonthly}{\myTodoLineGray}} -- my code doesn't work

Please help me.

**Reply by kudrykv** [reply]
That's weird.
I'll check that out; I will circle back tomorrow morning

**Reply by kudrykv** [reply]
@nmduc073, thanks for the waiting!
You also need to update `app/config/Config.go` — find the `Numbers` struct and put the `DailyNotesMonthly int` there, too.

**Reply by nmduc073** [reply]
I did:
In config.go, I added:
DailyNotes int
DailyNotesMonthly int --my code

However, last night I found out that the source from https://github.com/kudrykv/latex-yearly-planner/releases/tag/v0.0.8.2324 worked when I made a change. I am working on v0.0.8.2324.
